migrations/app/schema/20211202213138_index_user_roles_table.up.sql

```diff
@@ -0,0 +1,5 @@
+-- Add a database migration to index the user_id and deleted_at columns of the users_roles table.
+CREATE INDEX users_roles_user_id_idx ON users_roles(user_id);
```

Comment:
Small space change to make them be the same
```suggestion
CREATE INDEX users_roles_user_id_idx ON users_roles (user_id);
```

Reply:
Thanks Felipe! That table name gets me every time. I thought I made them all users_roles and missed one. When I make the changes, do I rerun the migrations? Or not?

Reply:
No problem. You can locally if you want, though in this case it'll just change a comment line. They haven't run in the deployed environments so those are fine.